.travis.yml

```diff
@@ -7,21 +7,20 @@ env:
   - LANG=en_US.UTF-8
   - PROJECT=ArrayRemove.xcodeproj
   - IOS_FRAMEWORK_SCHEME="ArrayRemove"
-  - IOS_SDK=iphonesimulator10.0
-  - EXAMPLE_SCHEME="iOS Example"
   matrix:
-    - DESTINATION="OS=9.3,name=iPhone 6"          SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES"
-    - DESTINATION="name=iPhone 7 Plus"     SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES"
+  - DESTINATION="OS=9.3,name=iPhone 6s"    SCHEME="$IOS_FRAMEWORK_SCHEME"  RUN_TESTS="YES" SIM_UUID="C335FB1B-868F-4E8C-B14A-3264B3C7F51D"
+  - DESTINATION="OS=10.1,name=iPhone 7"    SCHEME="$IOS_FRAMEWORK_SCHEME"  RUN_TESTS="YES" SIM_UUID="5F911B30-5F23-403B-9697-1DFDC24773C8"
+  - DESTINATION="OS=10.1,name=iPhone 7 Plus"    SCHEME="$IOS_FRAMEWORK_SCHEME"  RUN_TESTS="YES" SIM_UUID="E3B7D09E-DD71-4DAA-9D1D-6B678487D646"
```

Comment:
Since this supports Xcode 8.2, do you want to have one of the tests run on iOS 10.2?